gears/src/types/decimal256.rs

```diff
@@ -4,6 +4,8 @@ pub use cosmwasm_std::Decimal256;
 use cosmwasm_std::{StdError, Uint256};
 
 pub const ONE_DEC: Decimal256 = Decimal256::one();
+// 1/10^18
+pub const SMALLEST_DEC: Decimal256 = Decimal256::new(Uint256::from_u128(1u128));
```

Comment:
Why not use https://docs.rs/cosmwasm-std/latest/cosmwasm_std/struct.Decimal256.html#associatedconstant.MIN

Reply:
The `Decimal256::MIN` is 0. The sdk has another value that is written here. 

```rs
    let min = Decimal256::new(Uint256::from_u128(1u128));
    let min_2 = Decimal256::MIN;
    dbg!(&min, &min_2);
```
Output:
```
[src/main.rs:6:5] &min = Decimal256(0.000000000000000001)
[src/main.rs:6:5] &min_2 = Decimal256(0)
```

Reply:
Ah I think I understand. `SMALLEST_DEC` is the minimum positive value.